src/components/Hero.js

```diff
@@ -25,25 +27,87 @@ function Hero({handleLogIn}) {
           <MarketingButton primary onClick={handleLogIn}>
             Login
           </MarketingButton>
-          <a href="">
-            <MarketingButton>
-              <Octicon verticalAlign="middle" icon={getIconByName("book")} />
-              {" "}Read The Story
-            </MarketingButton>
-          </a>
+          <MarketingButton primary onClick={handleLogIn}>
+            Signup
+          </MarketingButton>
         </Wrapper>
       </SubHero>
       <TrustSection>
-        {/*
         <small>Trusted By</small>
         <ul>
-          <li>ONE</li>
-          <li>TWO</li>
-          <li>THREE</li>
-          <li>FOUR</li>
+          <li>GraphQL</li>
+          <li>NextJS</li>
+          <li>Electron</li>
+          <li>Golang</li>
         </ul>
-        */}
       </TrustSection>
+      <QuoteSection>
+        <Link to="/">
+          <img style={{height: 50}} alt="open sauced" src={logo} />
```

Comment:
Noticing weird squashing of the image on the page between mobile widths. Could be fixed with a `max-width: 150px`

![Screenshot 2020-05-24 22 31 30](https://user-images.githubusercontent.com/5713670/82781733-73a34780-9e0f-11ea-87bd-46b538f04cf1.png)

Reply:
Fixed